### PR TITLE
Improve homepage a bit, clean up old/unused code

### PR DESCRIPTION
**Pull request by j-f1**
- Improve layout of the homepage features section
- Make links more readable in dark mode
- Make the footer a little smaller and less empty feeling
- Add back usage of `<Key />` component to render keyboard shortcuts (as a bonus, it detects your OS and swaps cmd/ctrl)
- Remove Step and NextLink components. They never worked on this website.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -82,22 +82,16 @@ module.exports = {
...
-              {
-                label: 'Discord',
-                href: 'https://discord.gg/8JfAXSgfRf',
-              },
-            ],
+            label: 'GitHub',
+            href: 'https://github.com/actualbudget/actual',
+          },
+          {
+            label: 'Website Source',
```
[the remark this answers is not in the document]
Comment (j-f1) [reply docusaurus.config.js:93]: I believe that’s in your branch which has not merged yet

[the remark this answers is not in the document]
Comment (j-f1) [reply docusaurus.config.js:98]: (if you look below now it should be there)

[the remark this answers is not in the document]
Comment (rich-howell) [reply docusaurus.config.js:98]: > I believe that’s in your branch which has not merged yet

Really? I am having a real bad day 🤣

**Comment by rich-howell** [comment]
Looks really good, nice on @j-f1

**Pull request merged**